eknrenderer/eknr-boxed-set-info.h

```diff
@@ -0,0 +1,33 @@
+/* Copyright 2018 Endless Mobile, Inc. */
+
+#pragma once
+
+#if !(defined(_EKN_RENDERER_INSIDE_EKNR_H) || defined(COMPILING_EKNR))
+#error "Please do not include this header file directly."
+#endif
+
+#include <glib-object.h>
+
+G_BEGIN_DECLS
+
+typedef struct _EknrBoxedSetInfo {
```

Comment:
I realized we discussed this before libdmodel was a separate library. Now that it is merged to SDK master, would it be worth considering using `DmContent` and `DmSet` in this API? That would allow you to get rid of `EknrBoxedSetInfo` and simplify the `EknrRenderer` methods with 15 arguments.